### PR TITLE
nameChanges

### PR DESCRIPTION
**Pull request by Prathin67**

### CODE REVIEW

File: src/app/add/add.component.ts [hunk 1]
```diff
@@ -28,7 +28,7 @@ export class AddComponent implements OnInit {
   }
   submit(data:any){
 
-    this.serv.createELEMENT({...data,id:data['slno']}).subscribe(s =>{
+    this.serv.createELEMENT({...data,id:data['slno']}).subscribe(detail =>{
```
Comment (Sushmitha-Vk) [inline src/app/add/add.component.ts]: detail is not used inside subscribe so dont declare it

File: src/app/card/card.component.ts [hunk 2]
```diff
@@ -38,7 +38,7 @@ export class CardComponent {
     })
 
     dialogRef.afterClosed().subscribe(result=>{
```
Comment (Sushmitha-Vk) [inline src/app/card/card.component.ts]: same as above comment

**Review by Sushmitha-Vk — CHANGES_REQUESTED** [review]

File: src/app/add/add.component.ts [hunk 3]
```diff
@@ -28,10 +28,11 @@ export class AddComponent implements OnInit {
   }
   submit(data:any){
 
-    this.serv.createELEMENT({...data,id:data['slno']}).subscribe(detail =>{
-      window.location.reload();
+    this.serv.createELEMENT({...data,id:data['slno']}).subscribe({
+      
      
     })
+    window.location.reload();
```
Comment (Sushmitha-Vk) [inline src/app/add/add.component.ts]: this is wrong, `window.location.reload(); ` should be inside subscribe only. I just said to remove the value declaration

File: src/app/card/card.component.ts [hunk 4]
```diff
@@ -37,7 +37,7 @@ export class CardComponent {
 
     })
 
-    dialogRef.afterClosed().subscribe(result=>{
+    dialogRef.afterClosed().subscribe({
```
Comment (Sushmitha-Vk) [inline src/app/card/card.component.ts]: why is empty {} used here

File: src/app/menu-cmp/menu-cmp.component.ts [hunk 5]
```diff
@@ -77,10 +77,11 @@ export class MenuCmpComponent implements OnInit {
   }
   star(data:any){
     this.service.updatewish(data)
-    .subscribe(detail=>{
+    .subscribe({
+      
       
-      window.location.reload()
     })
+    window.location.reload()
```
Comment (Sushmitha-Vk) [inline src/app/menu-cmp/menu-cmp.component.ts]: same as above

**Review by Sushmitha-Vk — CHANGES_REQUESTED** [review]

File: src/app/card/card.component.ts (hunk 4 above)
Comment (Prathin67) [reply src/app/card/card.component.ts]: resolved

**Review by Sushmitha-Vk — APPROVED** [review]

**Pull request merged**